docs/getting-started/cash-app-pay.md

```diff
@@ -24,46 +24,46 @@ NEW (v4.3.0)
 {: .alert }
 > Cash App Pay is currently available in the following region(s): US
 
-Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK.
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App Pay Kit SDK.
```

Comment:
```suggestion
You can addd Cash App Pay support using your Afterpay merchant account. To do this, you must generate a token by sending a server-to-server call to the Afterpay API Create Checkout <!--can you provide a link to this here?--> endpoint with the parameter `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App Pay Kit SDK.
```

Reply:
Addressed in [3f0282c](https://github.com/afterpay/sdk-android/pull/185/commits/3f0282c492fe293fd098fd2f2b4b0912de2acf1e)

Reply:
In the first line, addd has three d's please remove one. sorry for missing this

Reply:
@ScottAntonacAP I think the first line needs to be revised, my understanding is we are wrapping CA SDK inside the AP SDK, when we say "support" it is confusing.

Reply:
We are not wrapping the Cash App Pay Kit SDK, but rather adding allowing the merchants app to support Cash App Pay using their Afterpay merchant account. That's why it needs to be imported. Happy to update the first line 🙂 but given that information, what would you suggest?

Reply:
Potentially the first line could just be changed to something like:

> You can add Cash App Pay support to your app using your Afterpay merchant account.

Reply:
With our latest enhancements, you can now support taking Cash App Pay payments using your Afterpay merchant account.  - does this sound better?

Reply:
You can support taking Cash App Pay payments using your Afterpay merchant account. or With our latest enhancements, you can now support taking Cash App Pay payments using your Afterpay merchant account.

Reply:
choose any one

Reply:
Addressed in [b469f13](https://github.com/afterpay/sdk-android/pull/185/commits/b469f1364f0069df608df43ababea3e33ac7bf5b)